src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioDocumentNavigationService.cs

```diff
@@ -317,9 +317,34 @@ private bool NavigateTo(ITextBuffer textBuffer, VsTextSpan vsTextSpan)
                     return false;
                 }
 
-                return ErrorHandler.Succeeded(
-                    textManager.NavigateToLineAndColumn2(
-                        vsTextBuffer, VSConstants.LOGVIEWID.TextView_guid, vsTextSpan.iStartLine, vsTextSpan.iStartIndex, vsTextSpan.iEndLine, vsTextSpan.iEndIndex, (uint)_VIEWFRAMETYPE.vftCodeWindow));
+                if (options.GetOption(NavigationOptions.PreferProvisionalTab))
```

Comment:
So this is copy/paste from the OpenDocument call earlier? Should we just be surrounding the calls to OpenDocument and NavigateTo with a single NewDocumentStateScope?

Reply:
oh ya. I somehow thought OpenDocument has public. but it is private. so. will do.